app/src/ui/blank-slate/create-tutorial-repository-dialog.tsx

```diff
@@ -204,8 +206,6 @@ export class CreateTutorialRepositoryDialog extends React.Component<
           )
         )
       }
-    } finally {
-      this.setState({ loading: false })
```

Comment:
I take it that doing this explicitly is no longer needed?

Reply:
i think this is no longer needed because `this.props.onDismissed()` is now guaranteed to be called _after_ the repo has been successfully created. calling `this.props.onDismissed()` will close this popup (unmounting the react component).